<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodePainter</class>
 <widget class="QWidget" name="NodePainter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HW-Node</string>
  </property>
  <property name="windowIcon">
   <iconset resource="hwlogo.qrc">
    <normaloff>:/new/prefix1/image/HWLogo.jpeg</normaloff>:/new/prefix1/image/HWLogo.jpeg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,10">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="3,1,1">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="pushButtonChooseFile">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditFileDir">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>选择节点</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="4">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonStaticDisplay">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>静态演示</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonDynamicDisplay">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>动态演示</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonContrast">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>对比</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="PaintChart" name="widgetCpu" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="PaintChart" name="widgetMem" native="true"/>
     </item>
     <item row="1" column="0">
      <widget class="PaintChart" name="widgetDisk" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="PaintChart" name="widgetNet" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintChart</class>
   <extends>QWidget</extends>
   <header location="global">paintchart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="hwlogo.qrc"/>
 </resources>
 <connections/>
</ui>
